src/Router.js

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Wrapper from './Components/Wrapper';
 import Login from './Pages/Login/Login';
+import SignupComplete from './Pages/SignupComplete/SignupComplete';
 
 const Router = () => {
   return (
     <BrowserRouter>
       <Wrapper>
         <Routes>
           <Route path="/" element={<Login />} />
+          <Route path="signupcomplete" element={<SignupComplete />} />
```

Comment:
- path 명확하게 작성해주시는 게 좋습니다.
- route 경로는 일반적으로 kebab-case로 작성합니다.
```suggestion
          <Route path="/signup-complete" element={<SignupComplete />} />
```

Reply:
넵 적용 완료했습니다.